.github/workflows/connect-test.yml

```diff
@@ -5,13 +5,15 @@ on:
   push:
     paths:
       - "packages/blockchain-link"
-      - "packages/connect/**"
       - "packages/connect-common/**"
       - "packages/connect-iframe/**"
       - "packages/connect-web/**"
+      - "packages/connect/**"
+      - "packages/protobuf"
```

Comment:
added `"packages/protobuf"` and `"packages/schema-utils"` and sorted alphabetically